lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -34,10 +36,10 @@ def fragment_cache(cached_hash, non_cached_hash)
         private
 
         ActiveModel.silence_warnings do
-          attr_reader :included, :fieldset
+          attr_reader :fieldset
         end
 
-        def serializable_hash_for_collection(serializer, options)
```

Comment:
Not required for this PR, but passing the serializer reference along is useless here as it is available via an `attr_reader`.